src/ray/protobuf/object_manager.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package ray.rpc;
+
+message ObjectInfo {
```

Comment:
Do we need to define `ObjectInfo` in pb? 
I think it's only used between plasma and raylet, right?